xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variablesInit.vm

```diff
@@ -261,6 +261,8 @@
   @state-danger-bg:                     #f8ecec;
   @state-success-bg:                    #e5f3df;
   @state-info-bg:                       #e1f1f9;
+  ## We want to keep overwritting the @brand-danger value from bootstrap as it does not pass contrast checks on #fff
```

Comment:
```suggestion
  ## We override the @brand-danger value from bootstrap as it does not pass contrast checks on #fff
```

There is at least a typo in "overwri**tt**ing", but for me, it is also not clear what this "want to keep" means, is there any reason to stop overriding this value that I missed?

Reply:
You're right, this formulation was too wordy and can be abridged :+1: